kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -8,17 +8,18 @@
 from multiprocessing.dummy import Pool as ThreadPool
 from threading import Lock
 
-from django.conf import settings
+from django.conf import settings as django_settings
```

Comment:
When should one use `from django.conf import settings` vs `from kalite import settings`? What is the recommended way for devs to access the user's settings module?